package/corrade/PKGBUILD

```diff
@@ -0,0 +1,50 @@
+# Author: williamjcm <w.jcm59@gmail.com>
+# Contributor: mosra <mosra@centrum.cz>
+# Based on the Corrade Arch Linux PKGBUILD maintained by xyproto and the MSYS2/MinGW PKGBUILD templates
+
+_realname=corrade
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=2018.10
+pkgrel=1
+pkgdesc='C++11/C++14 multiplatform utility library'
+provides=('corrade')
+arch=('any')
+url='https://magnum.graphics/corrade/'
+license=('MIT')
+makedepends=("${MINGW_PACKAGE_PREFIX}-cmake" 'git' "${MINGW_PACKAGE_PREFIX}-ninja")
+source=("${_realname}"::"git+https://github.com/mosra/corrade.git#tag=v$pkgver")
+sha1sums=('SKIP')
+
+build() {
+  [[ -d "${srcdir}"/build-${CARCH} ]] && rm -rf "${srcdir}"/build-${CARCH}
```

Comment:
Is removing the directory necessary? Personally I'm not doing that in any package and looking around, the MSYS [SDL2](https://github.com/Alexpux/MINGW-packages/blob/master/mingw-w64-SDL2/PKGBUILD) package doesn't do that either.

So unless that's required by some guidelines, I would remove this line :)

Reply:
I copy-pasted it from the [templates](https://github.com/Alexpux/MINGW-packages/tree/master/mingw-w64-PKGBUILD-templates).

Maybe they have a reason for doing that.

Reply:
Some Arch templates had that too, but from my experience 90% packaging errors were from something else than broken build directory. So rather than waiting every time to get it built from scratch I would wipe the `build-*` directory manually in those remaining 10%.

This is usually needed with venerable Makefile-based systems where dependencies are not rebuilt correctly and such, not much of a problem here.